Common/CommonFuncs.h

```diff
@@ -31,6 +31,8 @@
 
 #if defined(_M_IX86) || defined(_M_X86)
 #define Crash() {asm ("int $3");}
+#elif defined(HAVE_LIBNX)
+#define Crash() { while(1); };
```

Comment:
Hm, this seems decidedly not a crash... can we just use like `*(int *)0 = 0x1337;`?

-[Unknown]

Reply:
We can also install a full exception handler if needed + do a context restore, got it all ready in another project.
But yes, we can even throw a fatal, but that would cause a full reboot.